server/routes/api/comments.js

```diff
@@ -42,4 +43,8 @@ comments.patch('/comments/:commentId',
 comments.get('/comments/:commentId',
   verifyToken, validateResourceId, getComment);
 
+// delete a user comment
+comments.delete('/comments/:commentId',
+  verifyToken, validateResourceId, deleteComment);
```

Comment:
consider adding the `validateAccess` middleware before the `deleteComment` controller

Reply:
This is fixed now @augustineezinwa.

Reply:
This is fixed now @augustineezinwa.